charts/hono/README.md

```diff
@@ -666,10 +671,6 @@ The Helm chart can be configured to use these *native* images by means of explic
 name as described in *Using specific Container Images*. The names of the images based on native executables are the
 standard image names appended by `-native`.
 
-The native executables used in the images currently do not properly detect the memory and CPU resource limits defined on the
-container. This is due to a bug in the native-image builder in GraalVM versions before 22.1. Future versions of Hono will
-probably not be affected by this anymore.
```

Comment:
@sophokles73 I guess this can be removed as the [HonoBuilder.dockerfile](https://github.com/eclipse-hono/hono/blob/f7178e8b954d6bfc17724a3078ff7687d326df0d/jenkins/builder/HonoBuilder.dockerfile#L21) uses Mandrel/GraalVM version 23.0.2.1-Final now. Is that correct?